src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
```

Comment:
The function name `calculateRentalCost` is descriptive and clearly states its purpose.